packages/wrangler/src/triggers/index.ts

```diff
@@ -0,0 +1,88 @@
+import { findWranglerToml, readConfig } from "../config";
+import { getScriptName, isLegacyEnv, printWranglerBanner } from "../index";
+import * as metrics from "../metrics";
+import { requireAuth } from "../user";
+import triggersDeploy from "./deploy";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../yargs-types";
+
+export default function registerTriggersSubcommands(
+	triggersYargs: CommonYargsArgv
+) {
+	triggersYargs.command(
+		"deploy",
+		"Updates the triggers of your current deployment [beta]",
+		triggersDeployOptions,
+		triggersDeployHandler
+	);
+}
+
+export function triggersDeployOptions(yargs: CommonYargsArgv) {
+	return yargs
+		.option("name", {
+			describe: "Name of the worker",
+			type: "string",
+			requiresArg: true,
+		})
+		.option("triggers", {
+			describe: "cron schedules to attach",
+			alias: ["schedule", "schedules"],
+			type: "string",
+			requiresArg: true,
+			array: true,
+		})
+		.option("routes", {
+			describe: "Routes to upload",
+			alias: "route",
+			type: "string",
+			requiresArg: true,
+			array: true,
+		})
+		.option("dry-run", {
+			describe: "Don't actually deploy",
+			type: "boolean",
+		})
+		.option("legacy-env", {
+			type: "boolean",
+			describe: "Use legacy environments",
+			hidden: true,
+		})
+		.option("dispatch-namespace", {
+			describe:
+				"Name of a dispatch namespace to deploy the Worker to (Workers for Platforms)",
+			type: "string",
+		});
```

Comment:
I realise these options are copied from `deploy`, but this argument doesn't make sense for triggers

Reply:
Fixed in https://github.com/cloudflare/workers-sdk/pull/5436